sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -208,22 +208,22 @@ TEST_F(KeyVaultClientTest, CreateRsaHsmKey)
     auto rsaHsmKey = Azure::Security::KeyVault::Keys::CreateRsaKeyOptions(keyName, true);
     auto keyResponse = keyClient.CreateRsaKey(rsaHsmKey);
     CheckValidResponse(keyResponse);
-    auto keyVaultKey = keyResponse.ExtractValue();
+    auto keyVaultKey = keyResponse.Value;
     EXPECT_EQ(keyVaultKey.Name(), keyName);
   }
   {
     // Now get the key
     auto keyResponse = keyClient.GetKey(keyName);
     CheckValidResponse(keyResponse);
-    auto keyVaultKey = keyResponse.ExtractValue();
+    auto keyVaultKey = keyResponse.Value;
     EXPECT_EQ(keyVaultKey.Name(), keyName);
   }
   {
     // Delete key
     auto keyResponseOperation = keyClient.StartDeleteKey(keyName);
     auto keyResponse = keyResponseOperation.PollUntilDone(std::chrono::milliseconds(1000));
     CheckValidResponse(keyResponse);
-    auto keyVaultKey = keyResponse.ExtractValue();
+    auto keyVaultKey = keyResponse.Value;
     EXPECT_EQ(keyVaultKey.Name(), keyName);
   }
 }
```

Comment:
since you are here :)  would you mind?

Reply:
What would you like me to do?

Reply:
missing new line

Reply:
Ah gotcha. I wasn't seeing it because of differences between PR conversation view (which is what I typically use) and file view on GitHub:
![image](https://user-images.githubusercontent.com/6527137/112397132-ba2b3200-8cbe-11eb-8161-2309d8aba1ca.png)

Versus:
https://github.com/Azure/azure-sdk-for-cpp/pull/1974/files
![image](https://user-images.githubusercontent.com/6527137/112397151-c31c0380-8cbe-11eb-8a22-e9906200234c.png)